contracts/CallOptionToken.sol

```diff
@@ -47,8 +45,8 @@ contract CallOptionToken is ERC20, CloberOptionToken, ReentrancyGuard, Ownable {
     ) ERC20(name_, symbol_) {
         _underlyingToken = IERC20(underlyingToken_);
         _quoteToken = IERC20(quoteToken_);
-        _quotePrecisionComplement = 10**(18 - IERC20Metadata(quoteToken_).decimals());
-        _underlyingPrecisionComplement = 10**(18 - IERC20Metadata(underlyingToken_).decimals());
+        _precisionComplement =
+            10**(18 - IERC20Metadata(quoteToken_).decimals() + IERC20Metadata(underlyingToken_).decimals());
```

Comment:
```suggestion
            10**(18 + IERC20Metadata(underlyingToken_).decimals() - IERC20Metadata(quoteToken_).decimals());
```

Reply:
Also, `IERC20Metadata(underlyingToken_).decimals()` is called twice. We can reduce it to one call. See `PutOptionToken`.

Reply:
Resolved